### PR TITLE
Lvnod 23 Webpack configuration: dev and production.

### PR DESCRIPTION
**Pull request by AlbaMontis**

### CODE REVIEW

File: webpack/webpack.config.dev.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    loaders: []
+  },
+  plugins: [
+    new HtmlWebpackPlugin(({
+      title: 'DA-MI',
+      filename: 'index.html',
+      template: './app/src/component/template1.html'
+    }))
+  ]
+};
```
Comment (AlbaMontis) [inline webpack/webpack.config.dev.js]: Add end line

File: webpack/webpack.config.prod.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+var stripLoader = {
+ test: [/\.js$/],
+ exclude: /node_modules/,
+ loader: WebpackStripLoader.loader('console.log')
+}
+
+devConfig.module.loaders.push(stripLoader);
+
+module.exports = devConfig;
```
Comment (AlbaMontis) [inline webpack/webpack.config.prod.js]: Add end line

File: .babelrc [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ['es2015', 'react']
+}
```
Comment (denisbodnar) [inline .babelrc]: Add a new line

File: .gitignore [hunk 4]
```diff
@@ -1,37 +1,5 @@
...
-# Dependency directories
 node_modules
-jspm_packages
-
-# Optional npm cache directory
-.npm
 
-# Optional REPL history
-.node_repl_history
+build
```
Comment (denisbodnar) [inline .gitignore]: Add a new line

File: app/src/app.js [hunk 5]
```diff
@@ -0,0 +1,4 @@
+(() => console.log('HELLO WORLD!'))();
+
+
+import Hello from './component/hello.jsx';
```
Comment (denisbodnar) [inline app/src/app.js]: Add a new line

File: package.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    "html-webpack-plugin": "^2.28.0",
+    "strip-loader": "^0.1.2",
+    "webpack": "^2.5.1",
+    "webpack-dev-server": "^2.4.5"
+  },
+  "dependencies": {
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4"
+  }
+}
```
Comment (denisbodnar) [inline package.json:38]: Add a new line

File: app/src/component/template1.html [hunk 7]
```diff
@@ -0,0 +1 @@
+<div id='hello'></div>
```
Comment (denisbodnar) [inline app/src/component/template1.html]: Add a new line

File: app/src/component/hello.jsx [hunk 8]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+ 
+class Hello extends React.Component {
+  render() {
+    return <h1>Hello from React</h1>
+  }
+}
+ 
+ReactDOM.render(<Hello/>, document.getElementById('hello'));
```
Comment (denisbodnar) [inline app/src/component/hello.jsx]: Add a new line

**Review by denisbodnar — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

**Review by yarvanya — DISMISSED** [review]

**Review by yurii-kokhalevych — DISMISSED** [review]

File: app/src/app.js (hunk 5 above)
Comment (AlbaMontis) [reply app/src/app.js]: Done.

File: .gitignore (hunk 4 above)
Comment (AlbaMontis) [reply .gitignore]: Done.

File: package.json (hunk 6 above)
Comment (AlbaMontis) [reply package.json:38]: Done.

**Comment by AlbaMontis** [comment]
http://ssu-jira.softserveinc.com/browse/LVNOD-23

**Pull request closed without merging**